view/frontend/web/js/view/payment/method-renderer/profibro_paystack.js

```diff
@@ -6,16 +6,20 @@ define(
         'Magento_Checkout/js/view/payment/default',
         'Magento_Checkout/js/action/place-order',
         'Magento_Checkout/js/model/payment/additional-validators',
-        'Magento_Checkout/js/model/quote'
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/action/redirect-on-success'
```

Comment:
@AwoyoToyin I have backed up 0.9.13 as the version for 2.1.1 and below. Will merge the PR now